src/iosrv/srv.c

```diff
@@ -760,11 +763,21 @@ dss_start_xs_id(int xs_id)
 {
 	hwloc_obj_t	obj;
 	int		rc;
+	int		xs_core_offset;
+
+	/*
+	 * System XS all use the first core
+	 */
+	if (xs_id < dss_sys_xs_nr)
+		xs_core_offset = 0;
+	else
+		xs_core_offset = xs_id - dss_sys_xs_nr + 1;
```

Comment:
minor, "= xs_id - (dss_sys_xs_nr - DRPC_XS_NR)"  probably is a little better.

Reply:
Agreed, fixed.